### PR TITLE
feat(TFD1860) : Support multiple conditions in FilterRow

### PR DESCRIPTION
**Pull request by lbourgeois**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [X] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

Json stream for nested properities used by FilterRow commponent lacks those informations :
- min and max items for nested properties array
- ui:widget tag for nested widget

**What is the new behavior?**

Add those above missing informations

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/183/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/183/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/184/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/184/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/185/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/185/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/186/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/186/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/187/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/187/)

**Comment by sgandon** [comment]
@lbourgeois can you fill the PR description and check list please.

File: daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java [hunk 1]
```diff
@@ -158,6 +158,10 @@
         options.put(Widget.CODE_SYNTAX_WIDGET_CONF, "python");
         WIDGET_OPTIONS_MAPPING.put(Widget.CODE_WIDGET_TYPE, options);
 
+        options = new HashMap<>();
+        options.put("type", "default");
```
Comment (sgandon) [inline daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java]: I guess these option are specific for the NESTED_PROPERTIES widget, so please could you add the key "type" as a constant in the Widget class and explain in the NESTED_PROPERTIES javadoc how to use this option.

Comment (sgandon) [inline daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java]: Also I don't see where this option get used when converting the properties to json ??

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/188/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/188/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/190/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/190/)

File: daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java (hunk 1 above)
Comment (lbourgeois) [inline daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java]: This option is configured for the FilterRow component here : https://github.com/Talend/components/blob/feat/TFD1860_Properties_list_FilterRow/components/components-processing/processing-definition/src/main/java/org/talend/components/processing/definition/filterrow/FilterRowProperties.java#L82

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/191/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/191/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/192/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/192/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/193/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/193/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/197/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/197/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/198/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/198/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/199/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/199/)

File: daikon/src/test/java/org/talend/daikon/serialize/jsonschema/JsonSchemaUtilTest.java [hunk 2]
```diff
@@ -135,6 +137,17 @@ public static String readJson(String path) throws URISyntaxException, IOExceptio
         return new String(java.nio.file.Files.readAllBytes(resPath), "UTF8").trim();
     }
 
+    public static void writeJson(String content, String path) throws URISyntaxException, IOException {
```
Comment (RyanSkraba) [inline daikon/src/test/java/org/talend/daikon/serialize/jsonschema/JsonSchemaUtilTest.java]: Is this used someplace?

File: daikon/src/main/java/org/talend/daikon/properties/PropertiesList.java [hunk 3]
```diff
@@ -30,6 +30,13 @@
     public final static String ROW_NAME_PREFIX = "row";
 
     /**
+     * Min an d max items in the list
```
Comment (RyanSkraba) [inline daikon/src/main/java/org/talend/daikon/properties/PropertiesList.java]: typo

File: daikon/src/test/java/org/talend/daikon/serialize/jsonschema/JsonSchemaUtilTest.java (hunk 2 above)
Comment (lbourgeois) [inline daikon/src/test/java/org/talend/daikon/serialize/jsonschema/JsonSchemaUtilTest.java]: Only used for local debugging

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/200/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/200/)

**Pull request merged**